### PR TITLE
Add contributor + small tweaks

### PR DESCRIPTION
**Pull request by zextis**
- Add my name to contributors 
- Fix small errors in contributors file
- Add a podcast
- Adjust README syntax

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -1602,8 +1604,6 @@
 
 -[@useramrit](https://github.com/useramrit)
 
--[@AlexFil1983] (https://github.com/AlexFil1983)
```
Comment (timoerlemans) [inline CONTRIBUTORS.md]: Why did you remove this user?

Comment (zextis) [reply CONTRIBUTORS.md]: Check two or so lines below, it is a duplicate

Comment (timoerlemans) [reply CONTRIBUTORS.md]: You're right, sorry. :)
